bench/playbooks/roles/dns_caching/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-- name: Restart network manager
+- name: restart network manager
```

Comment:
Why?

Reply:
because below will cause the install process exit



- name: Add dnsmasq to network config
  lineinfile: >
    dest=/etc/NetworkManager/NetworkManager.conf
    regexp="dns="
    line="dns=dnsmasq"
    state=present
  when: result.stat.exists
  notify:
  - restart network manager

Reply:
This shouldn't make any difference in your setup. It's merely for [documentation/reference purpose](https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html).

Reply:
1584859181.448698-32385133842846/AnsiballZ_lineinfile.py && sleep 0’
<127.0.0.1> EXEC /bin/sh -c ‘rm -f -r /root/.ansible/tmp/ansible-tmp-1584859181.448698-32385133842846/ > /dev/null 2>&1 && sleep 0’
ERROR! The requested handler ‘restart network manager’ was not found in either the main handlers list nor in the listening handlers list
Traceback (most recent call last):
File “install.py”, line 437, in

Reply:
Above error gone after this fix.
This happened 12 months ago and no one fix it until today.

Reply:
yeah, I just checked and there was an issue with the title